docs/framework/unmanaged-api/profiling/icorprofilerinfo10-enumerateobjectreferences-method.md

```diff
@@ -37,7 +37,7 @@ HRESULT EnumerateObjectReferences( [in] ObjectID objectId,
  [in] Profiler provided data to pass to the `callback` function.
   
 ## Remarks  
- The `EnumerateObjectReferences` method is similar to [ObjectReferences](../../../../docs/framework/unmanaged-api/profiling/icorprofilerinfo-objectreferences-method.md), except that it walks the references one by one instead of forcing the profiler to pre-allocate an array to store the references in.  
+ The `EnumerateObjectReferences` method is similar to [ObjectReferences](../../../../docs/framework/unmanaged-api/profiling/icorprofilercallback-objectreferences-method.md), except that it walks the references on demand for the profiler instead of pre-allocating an array to store the references in.  
```

Comment:
I think you're looking at an older commit, I have it updated to 

> The `EnumerateObjectReferences` method is similar to [ObjectReferences](../../../../docs/framework/unmanaged-api/profiling/icorprofilercallback-objectreferences-method.md), except that it walks the references on demand for the profiler instead of pre-allocating an array to store the references. 

Let me know if that isn't sufficient